swift/project.yml

```diff
@@ -6,6 +6,7 @@ settings:
   base:
     HEADER_SEARCH_PATHS: $(SRCROOT)/wallet-core ${SRCROOT}/trezor-crypto/crypto
     SYSTEM_HEADER_SEARCH_PATHS: ${SRCROOT}/include ${SRCROOT}/../build/local/include ${SRCROOT}/trezor-crypto/include $(SRCROOT)/protobuf /usr/local/include /opt/homebrew/include
+    LIBRARY_SEARCH_PATHS: "$(inherited) ${SRCROOT}/../build/local/aarch64-apple-ios-sim/release"
```

Comment:
Need to find a conditional way to gen for simulator and for ios: https://gist.github.com/surpher/bbf88e191e9d1f01ab2e2bbb85f9b528#cross-compiling